addons/l10n_jo_edi/models/account_tax.py

```diff
@@ -0,0 +1,26 @@
+from odoo import models
+
+
+class AccountTax(models.Model):
+    _inherit = 'account.tax'
+
+    def _is_exempt_jo_tax(self):
+        self.ensure_one()
+        cid = self.company_id.id
+        exempted_taxes_refs = ['jo_zero_sale_exempted']
+        return self in [self.env.ref(f'account.{cid}_{tax_ref}') for tax_ref in exempted_taxes_refs]
```

Comment:
Use `self.env['chart.template'].with_company(self.company_id).ref('l10n_jo.jo_zero_sale_exempted')` instead.
Note the prefix `account` is wrong here. `jo_zero_sale_exempted` is not in the `account` module.